playbooks/roles/hammer_devel/tasks/hammer_config.yml

```diff
@@ -19,6 +19,20 @@
         :username: {{ hammer_devel_username }}
         :password: {{ hammer_devel_password }}
 
+- name: 'Configure hammer-cli-foreman-admin'
+  blockinfile:
+    dest: ~/.hammer/cli.modules.d/foreman_admin.yml
+    create: true
+    block: |
+      :foreman_admin:
+        :enable_module: true
```

Comment:
Yeah, that's a great idea - I was also thinking these files might just be copied from the hammer repository itself, since they all contain this simple config: https://github.com/Katello/hammer-cli-katello/tree/master/config That would allow future config changes to happen in the plugins rather than in Forklift.

Which would you prefer?